scripts/celery/calc.py

```diff
@@ -33,7 +35,7 @@ def iter_input(input_dir):
             basenames = os.listdir(path)
         except OSError as e:
             if e.errno == errno.ENOTDIR:
-                pass
+                continue
```

Comment:
I think this bug explains why in previous calculations there were duplicate jobs: `pass` means it would re-iterate through the previous value of `basenames`